### ISSUE TITLE
velocity affect mouseclick

### ISSUE THREAD

**Issue by michaelsteffen1982** [julianshapiro/velocity#888]
#### Your system information

* VelocityJS version: 2.0.5, 2.0.3
* Browser: Firefox, Chromium
* Operating System: Mint Mate 19, Raspbian

#### Checklist

* Is this an issue with *code*?: [Yes]
* Is this an issue with *documentation*?: [Maybe]
* Have you reproduced this in other browsers?: [Yes]
* Have you checked for updates?: [Yes]
* Have you checked for similar open issues?: [Yes]

> Please remember that this is an issue tracker, support requests should be posted on StackOverflow - see CONTRIBUTING.md

#### Please describe your issue in as much detail as possible:
I exchanged an jquery-fadeOut (preloader) with .velocity. Works fine with velocity 1.5.1. 
When I now exchange it to velocity 2.0.5 (+ loading script velocity.ui.min.js), I am not able to click some buttons on my hp anymore. The fade seem to work - but I cant click symbols and buttons anymore

#### Steps for reproducing this issue (code):

1. Not sure, I never used that fiddle-stuff and I am not sure what exactly causing this as the hp is made with a "construction-kit"

**Comment by Rycochet** [comment]
Velocity V2 has nothing to do with jQuery, so not sure what's going on there (it can make use of the jQuery selection stuff, but otherwise treats it just as an array of Elements). Definitely needs an example of this happening to be able to look into it at all.

**Comment by michaelsteffen1982** [comment]
If I would be able to reproduce this in JSFiddle, I possiblky would know whats wrong by myself...

But I coincidentally found out how to reproduce it with velocity 1.5.1 as I tried to workaround another problem I have:
Actually my page is using a simple preloader I found somewhere on the web, in the css-part its initiated with 

>     /* PRELOADER - CSS - requires JavaScript to work */
>         .no-js #loader { display: none; z-index: 1000; }
>         .js #loader { display: block; position: absolute; left: 100px; top: 0; z-index: 1000;}
>         .jtpl-logo { position: fixed; left: 0px;  top: 0px; padding: 75px 0; width: 100%; height: 100%; z-index: 1000; background: url() center no-repeat #fff}
> 

then scripts are loaded
```

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" type="text/javascript">
</script>

<script src="https://cdnjs.cloudflare.com/ajax/libs/velocity/1.5.1/velocity.min.js" type="text/javascript">
</script>

<script src="https://cdnjs.cloudflare.com/ajax/libs/slider-pro/1.4.0/js/jquery.sliderPro.min.js" type="text/javascript">
</script>
```
and in the Js-part
```
    // Wait until all images loaded
        window.onload=function() {
    // Fade
        $(".jtpl-logo").velocity("fadeOut", {duration: 400});  
};

```

And this works with 1.5.1.

Now I tried around a bit (I am a complete noob and not really know much what exactly I am doing doing) and added a "opacity: 1" to the Css-part of .jtpl-logo:

> .jtpl-logo { position: fixed; left: 0px;  top: 0px; padding: 75px 0; width: 100%; height: 100%; z-index: 1000; **opacity: 1**; background: url() center no-repeat #fff}

and changed the JS-Fade
> $(".jtpl-logo").velocity({ opacity: 0 }, 400);  

and have the same (buttons etc are non-clickable anymore) as when I exchange velocity 1.5.1 with 2.0.5 and using the working way with "fadeOut" / without opacity.

I not really know why, but I assume its not a bug, more a "I do something wrong with some kind of "layer-visibility" / display"...its just a bit strange to me as the "fadeOut"-way works with 1.5.1 and not with 2.0.5 anymore...maybe you can tell me what I am missing / doing wrong.


I stay on 1.5.1 anyway for a while as I found out that 2.* does not work anymore on my mobile and I love backward-compatibility.

**Comment by michaelsteffen1982** [comment]
Solved I think, it works with 1.5.1 and I assume it will with 2.*, too - if I add a display: none (like written somewhere on the homepage)
```
function Fader() {
        $(".jtpl-logo").velocity({ opacity: 0 }, { delay: 500, duration: 400, display: "none"});      
}; 
```

(still wondering a bit why fadeOut doesn't (with 2.*), but as said, I'll stay on 1.5 anyway and I assume it would work if I add a display:none)
